diff --git a/hunspellinputmethod_p.cpp b/hunspellinputmethod_p.cpp
index a5f3255..c6195c9 100644
--- a/hunspellinputmethod_p.cpp
+++ b/hunspellinputmethod_p.cpp
@@ -29,7 +29,7 @@

 #include <QtHunspellInputMethod/private/hunspellinputmethod_p_p.h>
 #include <QtVirtualKeyboard/qvirtualkeyboardinputcontext.h>
-#include <hunspell/hunspell.h>
+#include <hunspell.h>
 #include <QStringList>
 #include <QDir>
 #include <QTextCodec>
